<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTaskDialog</class>
 <widget class="QDialog" name="AddTaskDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Task</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Project:</string>
       </property>
       <property name="buddy">
        <cstring>projectLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="projectLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&amp;Title:</string>
       </property>
       <property name="buddy">
        <cstring>titleLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="titleLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>D&amp;escription:</string>
       </property>
       <property name="buddy">
        <cstring>descriptionTextEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPlainTextEdit" name="descriptionTextEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextEditorInteraction</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Status:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="statusNew">
         <property name="text">
          <string>&amp;New</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="statusStarted">
         <property name="text">
          <string>&amp;Started</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="statusDone">
         <property name="text">
          <string>&amp;Done</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="doneDateEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="dueDateLabel">
       <property name="text">
        <string>D&amp;ue Date:</string>
       </property>
       <property name="buddy">
        <cstring>dueDateEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="YDateEdit" name="dueDateEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="errorLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid red;
border-radius: 4px;
background-color: #fcc;
color: black</string>
     </property>
     <property name="text">
      <string notr="true">An error message</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>YDateEdit</class>
   <extends>QDateEdit</extends>
   <header>ydateedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>projectLineEdit</tabstop>
  <tabstop>titleLineEdit</tabstop>
  <tabstop>descriptionTextEdit</tabstop>
  <tabstop>dueDateEdit</tabstop>
  <tabstop>statusNew</tabstop>
  <tabstop>statusStarted</tabstop>
  <tabstop>statusDone</tabstop>
  <tabstop>doneDateEdit</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>statusDone</sender>
   <signal>toggled(bool)</signal>
   <receiver>doneDateEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddTaskDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>8</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddTaskDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
